opa/opa.go

```diff
@@ -189,3 +215,17 @@ func (opa *client) getDataQueryURL(path string) string {
 	}
 	return opa.getURL(fmt.Sprintf("v1/data/%s?%s", path, query))
 }
+
+func (opa *client) getOpaPackagePath(regoContent string) string {
```

Comment:
Maybe it's too expensive to do, but could this parse the policy and take the package from that?

```go
mod, _ := ast.ParseModule("foobar", regoContent)
return mod.Package
```

Reply:
also doesn't look like this needs to be a method, since `opa` isn't used here

Reply:
Fixed the method, but kept the regex for retrieving the package for now. The rules of this are basic enough that it may not be worth reparsing.
1. there can only be one package line or it won't compile
2. the line must begin with package